### PR TITLE
stability: add scheduler config.

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: stability-tester/config/scheduler.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	if conf.ShuffleRegion {
+		schedulerNames = append(schedulerNames, "shuffle-region-scheduler")
+	}
+
+	for {
+		for _, name := range schedulerNames {
+			data := map[string]string{"name": name}
+			b, _ := json.Marshal(data)
+			for _, addr := range conf.PDAddrs {
+				_, err := http.Post(addr+"pd/api/v1/schedulers", "application/json", bytes.NewBuffer(b))
```
Comment (siddontang) [inline stability-tester/config/scheduler.go:49]: if we add scheduler to the PD, do we need to add again in the for loop?

Comment (disksing) [reply stability-tester/config/scheduler.go:49]: stability constantly kills random pd-server, it uses a for loop to add schedulers after pd restarts.

**Review by siddontang — APPROVED** [review]
LGTM

**Comment by siddontang** [comment]
@cwen0 please add this to the stability test.

**Pull request merged**